core/src/main/java/bio/terra/pearl/core/service/notification/substitutors/EnrolleeEmailSubstitutor.java

```diff
@@ -58,6 +71,12 @@ public String lookup(String key) {
         return "";
     }
 
+    public String getSiteLink(PortalEnvironment portalEnv, Portal portal) {
+        String href = routingPaths.getParticipantBaseUrl(portalEnv, portal.getShortcode());
+        return String.format("<a rel=\"noopener\" href=\"%s\" target=\"_blank\">%s</a>", href, href);
```

Comment:
Should this link have some friendlier text than the URL?

Reply:
yeah, I waffled on that -- not sure what that text would be though -- I'll check if Erin has any quick thoughts